### PR TITLE
fix: [DHIS2-12304] show enrolllment errors in bottom snackbar

### PR DESCRIPTION
**Pull request by simonadomnisoru**

### CODE REVIEW

**Review by eirikhaugstulen — COMMENTED** [review]
Looks very good, @simonadomnisoru! Just one small question for you 👍

File: src/core_modules/capture-core/components/WidgetEnrollment/WidgetEnrollment.container.js [hunk 1]
```diff
@@ -8,16 +10,22 @@ import { useProgram } from './hooks/useProgram';
...
+        ownerOrgUnit,
+        enrollments,
+        refetch: refetchTEI,
+    } = useTrackedEntityInstances(teiId, programId);
     const { error: errorOrgUnit, displayName } = useOrganizationUnit(ownerOrgUnit);
     const canAddNew = enrollments.every(item => item.status !== plainStatus.ACTIVE);
+    const error = errorEnrollment || errorProgram || errorOwnerOrgUnit || errorOrgUnit;
+
+    if (error) {
+        log.error(errorCreator('Enrollment widget could not be loaded'));
```
[the remark this answers is not in the document]
Comment (simonadomnisoru) [reply src/core_modules/capture-core/components/WidgetEnrollment/WidgetEnrollment.container.js]: Good catch! I added the error. Thanks!

**Review by eirikhaugstulen — APPROVED** [review]
LGTM!

**Review by jasminenguyennn — APPROVED** [review]

**Review by geethaalwan — APPROVED** [review]
Tested successfully on 2.39 and 2.38.2 versions

**Pull request merged**